packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -1141,7 +1141,7 @@ class _MenuItemButtonState extends State<MenuItemButton> {
         trailingIcon: widget.trailingIcon,
         hasSubmenu: false,
         overflowAxis: _anchor?._orientation ?? widget.overflowAxis,
-        child: widget.child!,
+        child: widget.child ?? const SizedBox(),
```

Comment:
Using **const SizedBox.shrink();** Would be better. 
See https://api.flutter.dev/flutter/widgets/Visibility-class.html

Reply:
@mahmoudsaleh1997 Agreed, thank you